charts/trino/templates/configmap-catalog.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "trino.catalog" . }}
+  labels:
+    app: {{ template "trino.name" . }}
+    chart: {{ template "trino.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: catalogs
+data:
+  tpch.properties: |
+    connector.name=tpch
+    tpch.splits-per-node=4
```

Comment:
Why? I have never seen this config recommended anywhere yet?

Reply:
I think I just followed starburst example:
https://github.com/starburstdata/starburst-platform-charts/blob/master/starburst-enterprise/helm/values.yaml#L560
tbh I am not sure if it should be here.